Makefile

```diff
@@ -51,6 +51,24 @@ generate/go:
 
 	@echo Successfully compiled protos
 
+.PHONY: generate/ts
+generate/ts: description = Compile protobuf schema descriptor and generate types for Typescript
+generate/ts: clean/ts
+generate/ts:
+	docker run --platform linux/amd64 --rm -v ${PWD}:${PWD} -w ${PWD} ${PROTOC_IMAGE} \
```

Comment:
Do you need to `mkdir -p build/ts`?

Reply:
Actually, I think `/build/ts/descriptor-sets`?

Reply:
Yup, fixed!